### PR TITLE
added an option to contact app creators via email

### PR DESCRIPTION
**Pull request by iporat08**
Added an option of "contact us". 
On the screen in which an apartment searchers can modify its profile, I added a "contact us" button. 
When clicked the button opens a new activity in which the user is encouraged to send suggestions and\or complaints via email (the email is sent through the activity). 
Right now the complaints email address is my email address, you should change it to an email address to which complaints will be sent. 
The address is on line 21 in SendEmailActivity and is marked with TODO comment. 
Hope you'll like it!
Ido

### CODE REVIEW

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (amit-schwartz-u) [inline .idea/jarRepositories.xml:2]: is this file necessary?

File: .idea/.name [hunk 2]
```diff
@@ -0,0 +1 @@
+rooMe
```
Comment (amit-schwartz-u) [inline .idea/.name:1]: is this file necessary?

Comment (iporat08) [reply .idea/.name:1]: No, it was added by mistake - sorry!

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (iporat08) [reply .idea/jarRepositories.xml:2]: No, it was added by mistake - sorry!

**Review by amit-schwartz-u — APPROVED** [review]
ok I'll delete the unneeded files and move it to right location.. thanks a lot!
great feature!
